### DISCUSSION TITLE
Returned Emails By Mistake

### DISCUSSION THREAD

**Discussion by furiam91** [chromiecraft/chromiecraft#2347] [General]
Hello,

I just got to level 20 and got the gold in my mailbox, however, y pushed the return email button by mistake. Can anybody help me recover it? Please help me out!! :(

Any aid will be much appreciated.

Regards,

Furiam

**Comment by ghost** [comment]
Don't think so, find someone with money or get mining or herbing.
You got that in no time anyway...

**Reply by furiam91** [reply]
All righty, thanks anyway!
